### DISCUSSION TITLE
Connecting Pocket Computer to network

### DISCUSSION THREAD

**Discussion by Damino218** [MikaylaFischler/cc-mek-scada#493] [Support] [answered]
My pocket computer is stuck on the "Connecting to API" and I don't know what to do about it.

**Comment by MikaylaFischler** [comment]
Are all your network channels configured correctly and all your devices up-to-date? That would be the pocket channel, coordinator channel, and supervisor channel. The message you are getting is it trying to connect to the coordinator.

**Reply by Damino218** [reply]
My all channels are set to default.

**Reply by MikaylaFischler** [reply]
Please share your coordinator and pocket log.txt files from there respective computers

**Reply by Damino218** [reply]
[coordinator_log.txt](https://github.com/MikaylaFischler/cc-mek-scada/files/15443171/coordinator_log.txt)
[supervisor_log.txt](https://github.com/MikaylaFischler/cc-mek-scada/files/15443172/supervisor_log.txt)

**Reply by MikaylaFischler** [reply, marked as answer]
`[Sat May 25 11:08:14 2024] [INF] dropping API establish packet with incorrect api version v0.0.2 (expected v0.0.1)` per my prior comment, please update the coordinator (with `ccmsi update coordinator`)

**Reply by Damino218** [reply]
I wasn't able to update the coordinator, so I reinstalled it and now it works. Thank you for your help.

**Reply by MikaylaFischler** [reply]
Great, happy to hear that!
